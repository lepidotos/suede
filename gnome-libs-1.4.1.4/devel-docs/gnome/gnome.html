<HTML
><HEAD
><TITLE
>Gnome Library Reference Manual</TITLE
><META
NAME="GENERATOR"
CONTENT="Modular DocBook HTML Stylesheet Version 1.74b"><LINK
REL="NEXT"
TITLE="Gnome Library"
HREF="libgnome.html"></HEAD
><BODY
CLASS="BOOK"
BGCOLOR="#FFFFFF"
TEXT="#000000"
LINK="#0000FF"
VLINK="#840084"
ALINK="#0000FF"
><DIV
CLASS="BOOK"
><DIV
CLASS="TITLEPAGE"
><TABLE
WIDTH="100%"
BORDER="0"
BGCOLOR="#000000"
CELLPADDING="1"
CELLSPACING="0"
><TR
><TH
ALIGN="center"
VALIGN="center"
><FONT
COLOR="#FFFFFF"
SIZE="7"
><P
CLASS="TITLE"
><A
NAME="AEN2"
>Gnome Library Reference Manual</A
></P
></FONT
></TH
></TR
></TABLE
></DIV
><DIV
CLASS="TOC"
><DL
><DT
><B
>Table of Contents</B
></DT
><DT
><A
HREF="libgnome.html"
>Gnome Library</A
></DT
><DD
><DL
><DT
><A
HREF="gnome-gnome-config.html"
>gnome-config</A
>&nbsp;--&nbsp;Simple access to configuration values (50% done)</DT
><DT
><A
HREF="gnome-gnome-defs.html"
>gnome-defs</A
>&nbsp;--&nbsp;GNOME definitions for C++ linking.</DT
><DT
><A
HREF="gnome-gnome-dentry.html"
>gnome-dentry</A
>&nbsp;--&nbsp;Manipulation of .desktop files.</DT
><DT
><A
HREF="gnome-gnome-exec.html"
>gnome-exec</A
>&nbsp;--&nbsp;Execution of programs from GNOME applications</DT
><DT
><A
HREF="gnome-gnome-fileconvert.html"
>gnome-fileconvert</A
>&nbsp;--&nbsp;File format conversion functions.</DT
><DT
><A
HREF="gnome-gnome-help.html"
>gnome-help</A
>&nbsp;--&nbsp;Routines for displaying help</DT
><DT
><A
HREF="gnome-gnome-history.html"
>gnome-history</A
>&nbsp;--&nbsp;Keeping track of recently used documents.</DT
><DT
><A
HREF="gnome-gnome-i18n.html"
>gnome-i18n</A
>&nbsp;--&nbsp;Support for localization and internationalization.</DT
><DT
><A
HREF="gnome-gnome-metadata.html"
>gnome-metadata</A
>&nbsp;--&nbsp;File metadata information storage.</DT
><DT
><A
HREF="gnome-gnome-mime-info.html"
>gnome-mime-info</A
>&nbsp;--&nbsp;Routines to get information bound to a MIME type.</DT
><DT
><A
HREF="gnome-gnome-mime.html"
>gnome-mime</A
>&nbsp;--&nbsp;Routines to determine the MIME type of a file</DT
><DT
><A
HREF="gnome-gnome-paper.html"
>gnome-paper</A
>&nbsp;--&nbsp;Paper dimensions, printing unit conversions.</DT
><DT
><A
HREF="gnome-gnome-popt.html"
>gnome-popt</A
>&nbsp;--&nbsp;Command line argument parser</DT
><DT
><A
HREF="gnome-gnome-regex.html"
>gnome-regex</A
>&nbsp;--&nbsp;Regular expression cache implementation</DT
><DT
><A
HREF="gnome-gnome-remote.html"
>gnome-remote</A
>&nbsp;--&nbsp;User preference routines for executing commands on a remote host</DT
><DT
><A
HREF="gnome-gnome-score.html"
>gnome-score</A
>&nbsp;--&nbsp;Tracking score for games</DT
><DT
><A
HREF="gnome-gnome-sound.html"
>gnome-sound</A
>&nbsp;--&nbsp;Sound playing routines for GNOME applications</DT
><DT
><A
HREF="gnome-gnome-triggers.html"
>gnome-triggers</A
>&nbsp;--&nbsp;Hierarchical signal mechanism for application events.</DT
><DT
><A
HREF="gnome-gnome-url.html"
>gnome-url</A
>&nbsp;--&nbsp;Launch viewers for documents based on their URL</DT
><DT
><A
HREF="gnome-gnome-util.html"
>gnome-util</A
>&nbsp;--&nbsp;Miscellaneous utility functions for GNOME and C programs.</DT
><DT
><A
HREF="gnome-libgnome.html"
>libgnome</A
>&nbsp;--&nbsp;Non GUI part of the GNOME library.</DT
></DL
></DD
></DL
></DIV
></DIV
><DIV
CLASS="NAVFOOTER"
><BR
CLEAR="all"><BR><TABLE
WIDTH="100%"
BORDER="0"
BGCOLOR="#000000"
CELLPADDING="1"
CELLSPACING="0"
><TR
><TD
WIDTH="25%"
BGCOLOR="#C00000"
ALIGN="left"
>&nbsp;</TD
><TD
WIDTH="25%"
BGCOLOR="#0000C0"
ALIGN="center"
><FONT
COLOR="#FFFFFF"
SIZE="3"
><B
>&nbsp;</B
></FONT
></TD
><TD
WIDTH="25%"
BGCOLOR="#00C000"
ALIGN="center"
><FONT
COLOR="#FFFFFF"
SIZE="3"
><B
>&nbsp;</B
></FONT
></TD
><TD
WIDTH="25%"
BGCOLOR="#C00000"
ALIGN="right"
><A
HREF="libgnome.html"
><FONT
COLOR="#FFFFFF"
SIZE="3"
><B
>Next Page &#62;&#62;&#62;</B
></FONT
></A
></TD
></TR
><TR
><TD
COLSPAN="2"
ALIGN="left"
>&nbsp;</TD
><TD
COLSPAN="2"
ALIGN="right"
><FONT
COLOR="#FFFFFF"
SIZE="3"
><B
>Gnome Library</B
></FONT
></TD
></TR
></TABLE
></DIV
></BODY
></HTML
>