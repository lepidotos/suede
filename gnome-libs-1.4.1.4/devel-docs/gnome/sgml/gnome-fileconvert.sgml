<refentry id="gnome-gnome-fileconvert">
<refmeta>
<refentrytitle>gnome-fileconvert</refentrytitle>
<manvolnum>3</manvolnum>
<refmiscinfo>GNOME Library</refmiscinfo>
</refmeta>

<refnamediv>
<refname>gnome-fileconvert</refname><refpurpose>File format conversion functions.</refpurpose>
</refnamediv>

<refsynopsisdiv><title>Synopsis</title>
<synopsis>

#include &lt;gnome.h&gt;


<link linkend="gint">gint</link>        <link linkend="gnome-file-convert-fd">gnome_file_convert_fd</link>           (<link linkend="gint">gint</link> fd,
                                             <link linkend="gchar">gchar</link> *fromtype,
                                             <link linkend="gchar">gchar</link> *totype);
<link linkend="gint">gint</link>        <link linkend="gnome-file-convert">gnome_file_convert</link>              (const char *filename,
                                             const char *fromtype,
                                             const char *totype);
</synopsis>
</refsynopsisdiv>





<refsect1>
<title>Description</title>
<para>
These routines provide conversion of information based on a source and
target mime type.  They return -1 on failure, or an opened file
descriptor with the data on success</para>

<para>
The rules for converting from one MIME type to another are
loaded from the <filename>$gnome-prefix/share/type-convert</filename>
directory. Any files in that directory whose extension is
<filename>.convert</filename> is loaded as well as the per-user
<filename>~/.gnome/type.convert</filename> file.</para>

    <para>The file format used to define a filter is very
    simple:</para>

      <example>
	<title>Sample MIME type file conversion definition</title>
	<programlisting>
image/gif image/x-portable-pixmap giftopnm
	</programlisting>
     </example>
    <para>
	This sample file conversion file defines a conversion rule
	from the image/gif MIME type to the image/x-portable-pixmap
	MIME type.  The program used for the conversion is
	<filename>giftopnm</filename>.  This program should do the
	specified MIME type conversion by reading its standard input
	and writing the results on the standard output.
    </para>
</refsect1>

<refsect1>
<title>Details</title>
<refsect2>
<title><anchor id="gnome-file-convert-fd">gnome_file_convert_fd ()</title>
<programlisting><link linkend="gint">gint</link>        gnome_file_convert_fd           (<link linkend="gint">gint</link> fd,
                                             <link linkend="gchar">gchar</link> *fromtype,
                                             <link linkend="gchar">gchar</link> *totype);</programlisting>
<para>
Converts the file opened by <parameter>fd</parameter> to the <parameter>totype</parameter> format.</para>
<para>

</para><informaltable pgwide="1" frame="none" role="params">
<tgroup cols="2">
<colspec colwidth="2*">
<colspec colwidth="8*">
<tbody>
<row><entry align="right"><parameter>fd</parameter>&nbsp;:</entry>
<entry> file desciptor pointing to the file to convert.
</entry></row>
<row><entry align="right"><parameter>fromtype</parameter>&nbsp;:</entry>
<entry> mime type of the file
</entry></row>
<row><entry align="right"><parameter>totype</parameter>&nbsp;:</entry>
<entry>   target mime type we want
</entry></row>
<row><entry align="right"><emphasis>Returns</emphasis> :</entry><entry>-1 on failure, or a file descriptor to the converted
file.
</entry></row>
</tbody></tgroup></informaltable></refsect2>
<refsect2>
<title><anchor id="gnome-file-convert">gnome_file_convert ()</title>
<programlisting><link linkend="gint">gint</link>        gnome_file_convert              (const char *filename,
                                             const char *fromtype,
                                             const char *totype);</programlisting>
<para>
Converts <parameter>filename</parameter> to the <parameter>totype</parameter> format.</para>
<para>

</para><informaltable pgwide="1" frame="none" role="params">
<tgroup cols="2">
<colspec colwidth="2*">
<colspec colwidth="8*">
<tbody>
<row><entry align="right"><parameter>filename</parameter>&nbsp;:</entry>
<entry> file to convert
</entry></row>
<row><entry align="right"><parameter>fromtype</parameter>&nbsp;:</entry>
<entry> mime type of the file
</entry></row>
<row><entry align="right"><parameter>totype</parameter>&nbsp;:</entry>
<entry>   target mime type we want
</entry></row>
<row><entry align="right"><emphasis>Returns</emphasis> :</entry><entry>-1 on failure, or a file descriptor to the converted
file.
</entry></row>
</tbody></tgroup></informaltable></refsect2>

</refsect1>




</refentry>
