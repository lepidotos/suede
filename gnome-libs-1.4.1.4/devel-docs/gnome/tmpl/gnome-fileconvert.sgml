<!-- ##### SECTION Title ##### -->
gnome-fileconvert

<!-- ##### SECTION Short_Description ##### -->
File format conversion functions.

<!-- ##### SECTION Long_Description ##### -->
<para>
These routines provide conversion of information based on a source and
target mime type.  They return -1 on failure, or an opened file
descriptor with the data on success</para>

<para>
The rules for converting from one MIME type to another are
loaded from the <filename>$gnome-prefix/share/type-convert</filename>
directory. Any files in that directory whose extension is
<filename>.convert</filename> is loaded as well as the per-user
<filename>~/.gnome/type.convert</filename> file.</para>

    <para>The file format used to define a filter is very
    simple:</para>

      <example>
	<title>Sample MIME type file conversion definition</title>
	<programlisting>
image/gif image/x-portable-pixmap giftopnm
	</programlisting>
     </example>
    <para>
	This sample file conversion file defines a conversion rule
	from the image/gif MIME type to the image/x-portable-pixmap
	MIME type.  The program used for the conversion is
	<filename>giftopnm</filename>.  This program should do the
	specified MIME type conversion by reading its standard input
	and writing the results on the standard output.
    </para>

<!-- ##### SECTION See_Also ##### -->
<para>

</para>

<!-- ##### FUNCTION gnome_file_convert_fd ##### -->
<para>

</para>

@fd: 
@fromtype: 
@totype: 
@Returns: 


<!-- ##### FUNCTION gnome_file_convert ##### -->
<para>

</para>

@filename: 
@fromtype: 
@totype: 
@Returns: 


