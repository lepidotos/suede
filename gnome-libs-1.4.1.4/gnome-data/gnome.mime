#
#	NOTE: This file is on its way to obsolescence. GnomeVFS is
#	taking over MIME-handling responsibility, which will eventually
#	be eliminated from gnome-libs (after GnomeVFS ships).
#
#	GnomeVFS uses "gnome-vfs.mime" and explicitly ignores this
#	file if found. Nautilus uses GnomeVFS.
#
#	While this file is still around, if you make corrections, please
#	also correct gnome-vfs.mime.
#
#
# Copryght (C) The Free Software Foundation, 1998, 1999
# Copyright (C) 2000 Eazel, Inc.
# http://www.fsf.org
#
# Mime types as provided by the GNOME libraries for GNOME.
#
# Applications can provide more mime types by installing other
# .mime files in the $share/mime-info directory.
#
# The format of this file is:
#
# mime-type
#         ext[,prio]:   list of extensions for this mime-type
#         regex[,prio]: a regular expression that matches the filename
#
# more than one ext: and regex: fields can be present.
#
# prio is the priority for the match, the default is 1.  This is required 
# to distinguish composed filenames, for example .gz has a priority of 1
# and .tar.gz has a priority of 2 (thus a file having the filename
# something.tar.gz will match the mime-type for tar.gz before the mime-type
# for .gz
#

application/andrew-inset
	ext: ez

application/mspowerpoint
	ext: ppt

application/msword
	ext: doc

application/octet-stream
	ext: bin

application/oda
	ext: oda

application/pdf
	ext: pdf

application/pgp
	ext: pgp

application/postscript
	ext: ps eps

application/rtf
	ext: rtf

application/vnd.ms-excel
	ext: xls

application/x-applix-spreadsheet
        ext: as

application/x-applix-word
        ext: aw

application/x-arj
	ext: arj

application/x-asp
        ext: asp

application/x-backup
        ext: bak BAK
	
application/x-bcpio
	ext: bcpio

application/x-bzip
	ext: bz2 bz

application/x-bzip-compressed-tar
	regex,2: tar\.bz2$

application/x-cgi
        ext: cgi

application/x-chess-pgn
	ext: pgn

application/x-class-file
	ext: class

application/x-compressed-tar
	regex,2: tar\.gz$
	ext: tgz

application/x-core-file
	regex: core

application/x-cpio
	ext: cpio

application/x-cpio-compressed
	regex: \.cpio\.gz$

application/x-deb
	ext: deb

application/x-dvi
	ext: dvi

application/x-e-theme
        ext: etheme ETHEME

application/x-font-afm
	ext: afm

application/x-font-bdf
	ext: bdf

application/x-font-linux-psf
	ext: psf

application/x-font-pcf
	ext: pcf

application/x-font-speedo
	ext: spd

application/x-font-ttf
	ext: ttf TTF

application/x-font-type1
	ext: pfa pfb

application/x-gnome-app-info
	ext: desktop

application/x-gnumeric
	ext: gnumeric

application/x-gtar
	ext: gtar

application/x-gzip
	ext: gz

application/x-hdf
	ext: hdf

application/x-kde-app-info
	ext: kdelnk

application/x-lha
	ext: lha

application/x-lhz
	ext: lhz

application/x-mif
	ext: mif

application/x-netcdf
	ext: cdf nc

application/x-object-file
	ext: o

application/x-ogg
        ext: ogg

application/x-php
        ext: php php3 php4

application/x-profile
        regex: gmon\.out$

application/x-reject
	ext: rej
	
application/x-rpm
	ext: rpm

application/x-shar
	ext: shar

application/x-shared-library
	ext: so

application/x-shared-library-la
        ext: la

application/x-smil
	ext: smil smi sml

application/x-staroffice-presentation
	ext: sdd

application/x-staroffice-spreadsheet
	ext: sdc

application/x-staroffice-words
	ext: sdw

application/x-sv4cpio
	ext: sv4cpio

application/x-sv4crc
	ext: sv4crc

application/x-tar
	ext: tar

application/x-theme
        ext: theme

application/x-troff-man-compressed
	regex: ([^0-9]|^[^\.]*)\.([1-9][a-z]?|n)\.g?[Zz]$

application/x-unix-archive
	ext: a sa

application/x-ustar
	ext: ustar

application/x-wais-source
	ext: src

application/x-zoo
	ext: zoo

application/zip
	ext: zip



audio/basic
	ext: snd

audio/x-aiff
	ext: aif aifc aiff

audio/x-it
        ext: it IT

audio/x-midi
	ext: midi mid

audio/x-mod
	ext: mod MOD

audio/x-mp3
	ext: mp3

audio/x-mpegurl
        ext: m3u

audio/x-pn-realaudio
	ext: rm ra ram

audio/x-s3m
	ext: S3M s3m

audio/x-stm
	ext: STM stm

audio/x-ulaw
	ext: au

audio/x-voc
	ext: voc

audio/x-wav
	ext: wav

audio/x-xi
	ext: xi

audio/x-xm
	ext: xm XM



image/cgm
	ext: cgm

image/gif
	ext: gif

image/ief
	ext: ief

image/jpeg
	ext: jpe jpeg jpg

image/png
	ext: png

image/svg
	ext: svg

image/tiff
	ext: tif tiff

image/vnd.dwg
        ext: dwg

image/vnd.dxf
        ext: dxf

image/x-3ds
        ext: 3ds

image/x-applix-graphic
        ext: ag

image/x-bmp
	ext: bmp

image/x-cmu-raster
	ext: ras

image/x-compressed-xcf
	regex,2: \.xcf\.gz$
	regex.2: \.xcf\.bz2$

image/x-ico
	ext: ico

image/x-iff
        ext: iff IFF

image/x-ilbm
        ext: ilbm ILBM

image/x-lwo
        ext: lwo lwob

image/x-lws
        ext: lws

image/x-portable-anymap
	ext: pnm

image/x-portable-bitmap
	ext: pbm

image/x-portable-graymap
	ext: pgm

image/x-portable-pixmap
	ext: ppm

image/x-psd
	ext: psd

image/x-rgb
	ext: rgb

image/x-tga
	ext: tga

image/x-xbitmap
	ext: xbm

image/x-xcf
	ext: xcf

image/x-xfig
        ext: fig

image/x-xpixmap
	ext: xpm

image/x-xwindowdump
	ext: xwd



message/x-gnu-rmail
	regex: ^RMAIL$



model/vrml
	ext: wrl




text/bib
	ext: bib

text/css
	ext: css

text/html
	ext: html htm HTML

text/mathml
	ext: mml

text/plain
	ext: asc txt TXT

text/richtext
	ext: rtx

text/sgml
	ext: sgml sgm

text/tab-separated-values
	ext: tsv

text/x-authors
	regex: AUTHORS
	
text/x-c
	ext: c h

text/x-c++
	ext: cc C cpp c++ hp H h++

text/x-comma-separated-values
	ext: csv

text/x-copying
	regex: COPYING	

text/x-credits
	regex: CREDITS

text/x-csh
	ext: csh

text/x-dcl
	ext: dcl

text/x-dsl
	ext: dsl

text/x-dtd
	ext: dtd

text/x-emacs-lisp
	ext: el

text/x-fortran
	ext: f

text/x-gtkrc
        regex: [.|]gtkrc

text/x-idl
        ext: idl

text/x-install
	regex: INSTALL

text/x-java
	ext: java

text/x-makefile
	regex: [Mm]akefile

text/x-perl
	ext: pl perl

text/x-python
	ext: py

text/x-readme
	regex: README.*

text/x-scheme
	ext: scm

text/x-setext
	ext: etx

text/x-sh
	ext: sh

text/x-sql
	ext: sql

text/x-tcl
	ext: tcl

text/x-tex
	ext: tex

text/x-texinfo
	ext: texi texinfo

text/x-troff
	ext: roff t tr

text/x-troff-man
	regex: (([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|n)|\.man)$
	ext: man

text/x-troff-me
	ext: me

text/x-troff-ms
	ext: ms

text/x-vcalendar
	ext: vcs vcf

text/x-vcard
	ext: gcrd

text/xml
	ext: xml

video/mpeg
	ext: mp2 mpe mpeg mpg

video/quicktime
	ext: mov qt

video/x-anim
        ext: anim5 anim7

video/x-flc
	ext: flc

video/x-fli
	ext: fli

video/x-msvideo
	ext: avi

video/x-sgi-movie
	ext: movie
