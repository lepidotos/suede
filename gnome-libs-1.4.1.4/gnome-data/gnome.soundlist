[__section_info__]
description=GNOME system events
description[ca]=Events de sistema de GNOME
description[da]=Systemhændelser i Gnome skrivebordsmiljøet
description[de]=Sound-Ereignisse für GNOME
description[el]=Ãåãïíüôá ôïõ óõóôŞìáôïò GNOME
description[es]=Eventos sistema de GNOME
description[et]=GNOME süsteemsed sündmused
description[fi]=GNOMEn järjestelmätapahtumat
description[fr]=Événements système de GNOME
description[gl]=Eventos de sistema de GNOME
description[it]=Eventi di sistema GNOME
description[ko]=±×³ğ ½Ã½ºÅÛ ÀÌº¥Æ®
description[lt]=GNOME sistemos ávykiai
description[no]=Systemhendelser for GNOME
description[pl]=Zdarzenia systemowe GNOME
description[pt]=Eventos de sistema GNOME
description[sk]=Systémové udalosti GNOME
description[tr]=Gnome sistem eylemleri
description[uk]=óÉÓÔÅÍÎ¦ ĞÏÄ¦§ GNOME
description[wa]=Evenmints sistinme di GNOME
description[zh_TW]=GNOME ¨t²Î¨Æ¥ó

[login]
file=login.wav
description=Login
description[ca]=Entrada al sistema
description[da]=Indlogning
description[de]=Anmeldung zur Sitzung
description[el]=¸íáñîç óõíåäñßáò
description[es]=Apertura de sesión
description[et]=Sisselogimine
description[fi]=Sisään kirjautuminen
description[fr]=Ouverture de session
description[gl]=Comezo de sesión
description[it]=Login
description[ko]=·Î±×ÀÎ
description[lt]=Prisijungimas
description[no]=Innlogging
description[pl]=Rozpoczêcie pracy
description[pt]=Início de Sessão
description[sk]=Prihlásenie
description[tr]=Giriş
description[uk]=÷È¦Ä Õ ÓÉÓÔÅÍÕ
description[wa]=Intreye è session
description[zh_TW]=µn¤J

[logout]
file=logout.wav
description=Logout
description[ca]=Eixida del sistema
description[da]=Udlogning
description[de]=Abmeldung von der Sitzung
description[el]=¸îïäïò áğü ôç óõíåäñßá
description[es]=Salida de sesión
description[et]=Väljalogimine
description[fi]=Ulos kirjautuminen
description[fr]=Fin de session
description[gl]=Fin de sesión
description[it]=Esci da GNOME
description[ko]=·Î±×¾Æ¿ô
description[lt]=Atsijungimas
description[no]=Utlogging
description[pl]=Zakoñczenie pracy
description[pt]=Fim de Sessão
description[sk]=Odhlásenie
description[tr]=Çıkış
description[uk]=úÁ×ÅÒÛÅÎÎÑ ÒÏÂÏÔÉ
description[wa]=Cwand li session est hûte
description[zh_TW]=µn¥X

[info]
file=info.wav
description=Informational Messages
description[ca]=Missatges informatius
description[da]=Informative beskeder
description[de]=Informationen
description[el]=Mçíıìáôá Ğëçñïöüñçóçò
description[es]=Mensajes informativos
description[et]=Teated
description[fi]=Tiedoteviestit
description[fr]=Messages d'information
description[gl]=Mensaxes de información
description[it]=Informazioni
description[ko]=¾Ë¸² ¸Ş¼¼Áö
description[lt]=Informacinës şinutës
description[no]=Informative meldinger
description[pl]=Komunikat informacyjny
description[pt]=Mensagens Informativas
description[sk]=Informácie
description[tr]=Bilgi iletileri
description[uk]=¶ÎÆÏÒÍÁÃ¦ÊÎ¦ ĞÏ×¦ÄÏÍÌÅÎÎÑ
description[wa]=Messaedjes d' informåcion
description[zh_TW]=¸ê®Æ°T®§

[warning]
file=warning.wav
description=Warning Messages
description[ca]=Missatges d'avís
description[da]=Advarsler
description[de]=Warnungen
description[el]=Ìçíıìáôá ĞñïåéäïğïéŞóåùí
description[es]=Mensajes de advertencia
description[et]=Hoiatused
description[fi]=Varoitusviestit
description[fr]=Messages d'avertissement
description[gl]=Mensaxes de aviso
description[it]=Avviso
description[ko]=°æ°í ¸Ş¼¼Áö
description[lt]=Áspëjamosios şinutës
description[no]=Advarsler
description[pl]=Komunikat ostrzegawczy
description[pt]=Avisos
description[sk]=Varovania
description[tr]=Uyarı iletileri
description[uk]=ğÏĞÅÒÅÄÖÕ×ÁÌØÎ¦ ĞÏ×¦ÄÏÍÌÅÎÎÑ
description[wa]=Messaedjes d' avertixhmint
description[zh_TW]=Äµ§i°T®§

[error]
file=error.wav
description=Error Messages
description[ca]=Missatges d'error
description[da]=Fejlmeldinger
description[de]=Fehler
description[el]=Ìçíıìáôá óöÜëìáôïò
description[es]=Mensajes de error
description[et]=Veateated
description[fi]=Virheviestit
description[fr]=Messages d'erreur
description[gl]=Mensaxes de erro
description[it]=Errore
description[ko]=¿À·ù ¸Ş¼¼Áö
description[lt]=Klaidø praneğimai
description[no]=Feilmeldinger
description[pl]=Komunikat b³êdu
description[pt]=Mensagens de Erro
description[sk]=Chyby
description[tr]=Hata iletileri
description[uk]=ğÏ×¦ÄÏÍÌÅÎÎÑ ĞÒÏ ĞÏÍÉÌËÕ
description[wa]=Messaedjes d' aroke
description[zh_TW]=¿ù»~°T®§

[question]
file=question.wav
description=Question Dialogs
description[ca]=Diàlegs interrogatius
description[da]=Spørgsmålsdialoger
description[de]=Fragen
description[el]=ÄéÜëïãïé ìå åñùôŞóåéò
description[es]=Diálogos con preguntas
description[et]=Küsimused
description[fi]=Kysymysvalintaikkunat
description[fr]=Dialogues avec des questions
description[gl]=Diálogos con preguntas
description[it]=Domanda
description[ko]=¹°À½ ´ëÈ­»óÀÚ
description[lt]=Klausimø dialogai
description[no]=Spørsmålsdialoger
description[pl]=Komunikat z zapytaniem
description[pt]=Diálogos com questões
description[sk]=Otázky
description[tr]=Sorgulama iletileri
description[uk]=ä¦ÁÌÏÇÉ ¦Ú ÚÁĞÉÔÁÎÎÑÍÉ
description[wa]=Purneas di dialogue avou des kesses
description[zh_TW]=¸ß°İ¹ï¸Üµ¡

[generic]
file=generic.wav
description=Miscellaneous Messages
description[ca]=Missatges diversos
description[da]=Diverse beskeder
description[de]=Sonstige Meldungen
description[el]=ÄéÜöïñá ìçíıìáôá
description[es]=Mensajes misceláneos
description[et]=Muud teated
description[fi]=Sekalaiset viestit
description[fr]=Autres messages
description[gl]=Outras mensaxes
description[it]=Evento generico
description[ko]=±× ¿Ü ¸Ş¼¼Áö
description[lt]=Ávairios şinutës
description[no]=Forskjellige beskjeder
description[pl]=Ró¿ne komunikaty
description[pt]=Outras mensagens
description[sk]=Rôzne správy
description[tr]=Diğer iletiler
description[uk]=ò¦ÚÎ¦ ĞÏ×¦ÄÏÍÌÅÎÎÑ
description[wa]=Ôtes messaedjes
description[zh_TW]=¨ä¥¦°T®§
