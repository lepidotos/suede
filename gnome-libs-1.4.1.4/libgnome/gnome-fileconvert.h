#ifndef __GNOME_FILECONVERT_H__
#define __GNOME_FILECONVERT_H__ 1

/* WARNING ____ IMMATURE API ____ liable to change */

BEGIN_GNOME_DECLS

/* Returns -1 if no conversion is possible */
gint
gnome_file_convert_fd(gint fd, const char *fromtype, const char *totype);
/* Convenience wrapper for the above function */
gint
gnome_file_convert(const char *filename, const char *fromtype, const char *totype);

END_GNOME_DECLS

#endif /* __GNOME_FILECONVERT_H__ */
