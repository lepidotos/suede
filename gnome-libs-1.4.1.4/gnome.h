#ifndef GNOME_H
#define GNOME_H

#if defined(GNOME_INTERFACE_VERSION)
#if GNOME_INTERFACE_VERSION > 1
#error "You are picking up an old header file because of a bad include path."
#error "Please make sure `gnome-config --cflags ...` output is before"
#error "any other -I options in your compilation flags."
#endif
#endif

#include "gnomesupport.h"

#include <gtk/gtk.h>
#include <gdk_imlib.h>
#include "libgnome/libgnome.h"
#include "libgnomeui/libgnomeui.h"
#endif
