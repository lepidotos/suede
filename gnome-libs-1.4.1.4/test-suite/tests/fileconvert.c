#include <gnome.h>
#include <libgnome/gnome-fileconvert.h>
#include <glib.h>
#include <stdio.h>
#include <unistd.h>
#include <sys/types.h>
#include <sys/stat.h>

/* Do whatever you need to do... You should put the "expected" output of
   the program in ../expected/testname.out, which will be compared against
   the actual output of the test when it is run.

   A non-zero exit code also indicates failure.
*/

int main(int argc, char *argv[])
{
	gint outfd, readlen = 0;
	gchar abuf[128];
	gchar *fn;
	
	gnomelib_init("fileconvert", "0.0");
	fn = g_strconcat(getenv("srcdir"), "/tests", NULL);
	chdir(fn);
	chmod("lynxdump.sh", S_IXGRP|S_IXOTH|S_IXUSR|S_IRGRP|S_IROTH|S_IRUSR|S_IWUSR);
	outfd = gnome_file_convert("fileconvert.in",
				   "text/html",
				   "text/ascii");
	if(outfd >= 0) return 1;
	outfd = gnome_file_convert("fileconvert.in",
				   "text/html",
				   "text/plain");
	if(outfd < 0) return 1;
	while((readlen = read(outfd, abuf, sizeof(abuf) - 1))) {
		printf("%.*s", readlen, abuf); fflush(stdout);
	} 

	printf("\n");
	fflush(stdout); /* Make sure to fflush things after you do
			   any output, so what you see on your terminal
			   is the same as what goes into the output file */
	return 0;
}
