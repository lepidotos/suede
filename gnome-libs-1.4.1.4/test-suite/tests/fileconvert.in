<html>
 <head>
  <title>Test Page</title>
 </head>
 <body>
This is a <a href="#foo">test</a> of the file conversion routines<a
name="foo">.</a>
 </body>
</html>
