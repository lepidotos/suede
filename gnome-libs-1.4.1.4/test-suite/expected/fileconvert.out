
   This is a [1]test of the file conversion routines.


References

   1. file://localhost/THISFILE.html#foo

Exit code 0
